spec.html

```diff
@@ -26820,7 +26820,7 @@ <h1>Example Cyclic Module Record Graphs</h1>
               </tr>
               <tr>
                 <th>_D_</th>
-                <td>3</td>
+                <td>2</td>
```

Comment:
We also need to correct the C entry just before this one.

Reply:
This PR does it :P

Reply:
Not on line 26814...